RadixWallet/Features/SettingsFeature/DebugSettings/Coordinator/DebugSettingsCoordinator+View.swift

```diff
@@ -89,77 +81,94 @@ extension DebugSettingsCoordinator.View {
 
 // MARK: - Extensions
 
+private extension StoreOf<DebugSettingsCoordinator> {
+	var destination: PresentationStoreOf<DebugSettingsCoordinator.Destination> {
+		func scopeState(state: State) -> PresentationState<DebugSettingsCoordinator.Destination.State> {
+			state.$destination
+		}
+		return scope(state: scopeState, action: Action.destination)
+	}
+}
+
+@MainActor
 private extension View {
-	@MainActor
-	func debugUserDefaultsContents(
-		with destinationStore: PresentationStoreOf<DebugSettingsCoordinator.Destinations>
+	func destinations(with store: StoreOf<DebugSettingsCoordinator>) -> some View {
+		let destinationStore = store.destination
+		return factorSources(with: destinationStore)
+			.debugUserDefaultsContents(with: destinationStore)
+		#if DEBUG
+			.debugKeychainTest(with: destinationStore)
+			.debugKeychainContents(with: destinationStore)
+		#endif
+			.debugInspectProfile(with: destinationStore)
+			.securityStructureConfigs(with: destinationStore)
+	}
+
+	private func debugUserDefaultsContents(
+		with destinationStore: PresentationStoreOf<DebugSettingsCoordinator.Destination>
 	) -> some View {
 		navigationDestination(
 			store: destinationStore,
-			state: /DebugSettingsCoordinator.Destinations.State.debugUserDefaultsContents,
-			action: DebugSettingsCoordinator.Destinations.Action.debugUserDefaultsContents,
+			state: /DebugSettingsCoordinator.Destination.State.debugUserDefaultsContents,
+			action: DebugSettingsCoordinator.Destination.Action.debugUserDefaultsContents,
 			destination: { DebugUserDefaultsContents.View(store: $0) }
 		)
 	}
 
 	#if DEBUG
-	@MainActor
-	func debugKeychainTest(
-		with destinationStore: PresentationStoreOf<DebugSettingsCoordinator.Destinations>
+	private func debugKeychainTest(
+		with destinationStore: PresentationStoreOf<DebugSettingsCoordinator.Destination>
 	) -> some View {
 		navigationDestination(
 			store: destinationStore,
-			state: /DebugSettingsCoordinator.Destinations.State.debugKeychainTest,
-			action: DebugSettingsCoordinator.Destinations.Action.debugKeychainTest,
+			state: /DebugSettingsCoordinator.Destination.State.debugKeychainTest,
+			action: DebugSettingsCoordinator.Destination.Action.debugKeychainTest,
 			destination: { DebugKeychainTest.View(store: $0) }
 		)
 	}
 
 	@MainActor
 	func debugKeychainContents(
-		with destinationStore: PresentationStoreOf<DebugSettingsCoordinator.Destinations>
+		with destinationStore: PresentationStoreOf<DebugSettingsCoordinator.Destination>
 	) -> some View {
 		navigationDestination(
 			store: destinationStore,
-			state: /DebugSettingsCoordinator.Destinations.State.debugKeychainContents,
-			action: DebugSettingsCoordinator.Destinations.Action.debugKeychainContents,
+			state: /DebugSettingsCoordinator.Destination.State.debugKeychainContents,
```

Comment:
Thanks for fixing the Git Merge Conflict 🙏

Reply:
It wasn't bad at all, I just chose your changes and then applied my changes to those files afterwards